my-app/src/main/java/com/mycompany/app/findInverseSquareRoot.java

```diff
@@ -0,0 +1,26 @@
+package com.mycompany.app;
+
+public class findInverseSquareRoot {
```

Comment:
Class names should be capitalized.

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Done